packages/web-components/src/horizontalbarchart/horizontalbarchart.stories.ts

```diff
@@ -95,6 +95,97 @@ const singleBarHBCData = [
   },
 ];
 
+const singleBarNMVariantData = [
+  {
+    chartTitle: 'one',
```

Comment:
create 1 more story with a single datapoint

Reply:
Okay

Reply:
Added. You can review once and then we can merge this PR.